bigbluebutton-html5/imports/ui/components/video-preview/component.jsx

```diff
@@ -188,6 +190,8 @@ class VideoPreview extends Component {
       deviceError: null,
       previewError: null,
     };
+
+    this.mirrorOwnWebcam = VideoService.mirrorOwnWebcam(props.user);
```

Comment:
I think it could be moved to state

Reply:
See above for an explanation why it doesn't belong in state. It could be moved to state if we implement a feature where the user can toggle this during the session. For now, this is sufficient in my opinion.